client/app/components/dashboards/gridstack/index.js

```diff
@@ -121,67 +122,6 @@ function gridstack($parse, dashboardGridOptions) {
         });
       };
 
-      this.batchUpdateWidgets = (items) => {
```

Comment:
@kravets-levko I removed `batchUpdateWidgets` cause it's not relevant anymore, but I didn't touch `batchUpdateAssignable` cause I'm not certain what it does. Can I remove it as well?

Reply:
IIRC, it was used for `Cancel` button (to move widgets back to original positions). If you removed that functionality - you can remove this as well.

Reply:
It's where it was used: https://github.com/getredash/redash/blob/fe4a7b65e70b356989a330fea3e17993216f5997/client/app/pages/dashboards/dashboard.html#L94

Reply:
Oh now I remember, I was actually afraid of touching `isOneColumnModeAssignable` - what's that used for?

Reply:
It's updated by `gridstack` component - dashboard controller need to know when grid is in single-column mode to update UI (e.g. disable Edit menu item). You can find all related code in `components/gridstack/index.js` and `pages/dashboards/dashboard.js/.html`

Reply:
Got it. Updated the code. Removed some more related stuff.

Tested:
1. Resizing to 1 col mode and back disables/enables the "Done editing" button.
2. Resizing to 1 col mode and back hides/shows the "Edit" menu item.
3. Public dashboard ok.